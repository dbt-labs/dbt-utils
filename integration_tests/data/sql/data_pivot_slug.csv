size,color
S,Red Color
S,Blue Color
M,Red Color
