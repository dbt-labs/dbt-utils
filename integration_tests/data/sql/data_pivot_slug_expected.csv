size,red_color,blue_color
S,1,1
M,1,0
